docs/antora.yml

```
@@ -1,10 +1,15 @@
 name: docs
 title: Payara Cloud
 version: master
-display_version: '21.11.0'
+display_version: 'June 2022'
 start_page: ROOT:Overview.adoc
 nav:
-- modules/ROOT/nav.adoc
+  - modules/ROOT/nav.adoc
+  - modules/cli/nav.adoc
 asciidoc:
   attributes:
     currentVersion: '1.0'
+    experimental: true
```

True values are represented with a empty string in antora
```suggestion
    experimental: ''
```

---

The experimental attribute should be defined in the playbook, otherwise it will set a variable 'experimental' to string value 'true'

---

Attributes defined in playbook or antorra.yml have the same effect per my understanding -- both set the attributes onto all documents. Therefore I prefer to have them at smallest scope, as production playbook lives elsewhere.